relayer/naive-strategy.go

```diff
@@ -200,10 +211,19 @@ func (nrs *NaiveStrategy) RelayPacketsOrderedChan(src, dst *Chain, sp *RelaySequ
 		} else {
 			msgs.Src = append(msgs.Src, msg)
 		}
+
+		txSize += len(msg.GetSignBytes())
```

Comment:
This is not the right calculation right? Should be the marshalled message right?

Reply:
GetSignBytes returns the marshalled message that is signed over. What is missing in this check is the addition of the entire transaction being marshalled, including msg sigs. I think adding that check would be difficult because after adding each message, you'd have to construct the tx and see if it is too big. I think I should just change the naming to be max msg size restricting the total length of the messages included in the tx